mm2src/coins/eth.rs

```diff
@@ -1243,55 +1245,348 @@ impl SwapOps for EthCoin {
         MmError::err(ValidateInstructionsErr::UnsupportedCoin(self.ticker().to_string()))
     }
 
-    fn is_supported_by_watchers(&self) -> bool { false }
+    fn is_supported_by_watchers(&self) -> bool { true }
```

Comment:
Q: Will this function have work flow in the future? If not, we don't need to register functions, it's better to have this functionality via coins configurations.

Reply:
Good point, I was going to remove it and use the coins config, but it's better to keep it the same way then

Reply:
> supports_watchers

Don't we use `unwrap_or(DEFAULT_VALUE)` approach while parsing the config anyway?

Reply:
I believe all coins should support watchers in the future, so adding if a coin supports watchers to config will be an overkill, since we will add it to all or most coins configs eventually. We can implement a default implementation
```rust
fn is_supported_by_watchers(&self) -> bool { false }
```
in `lp_coins.rs` and implement it to `true` for Eth and Utxo only for now.